### ISSUE TITLE
Restrict loop accessibility to new participants

### ISSUE THREAD

**Issue by giuliamml** [CollActionteam/clothing-loop#239]
Currently, there's no limit to the number of participants. However, the chainAdmin should be able to manage its own loop's accessibility and not allow new participants to join (while maintaining the loop visible on the map). The feature works as follows:

- The accessibility should be toggled through a switcher on the admin page (which can replicate the behaviour of the 'visible/invisible' switcher - see img attached)
- Once the switcher is set to 'close', new participants won't be able to join that loop and so the _'join'_ button in the popup on `FindChain.tsx` gets disabled, greyed out and onHover displays a popover with the following text: ~~"This Loop is currently not taking new participants. Click here (as link to "/loops/new/users/signup") to start your own"~~
"This Loop is at capacity currently and therefore not accepting new participants. Please check back in later, or _start your own Loop_! (link to "/loops/new/users/signup")"

![Screenshot 2022-05-23 at 15 43 39](https://user-images.githubusercontent.com/52887737/169849066-114805ba-569f-4b9d-bcc2-fdc9a5367af8.png)
![Screenshot 2022-05-23 at 15 45 36](https://user-images.githubusercontent.com/52887737/169849095-486de57b-77cc-4382-a556-d4e096d5a305.png)
